src/OrchardCore/OrchardCore.Data.Abstractions/IDataMigrationManager.cs

```diff
@@ -28,7 +30,8 @@ public interface IDataMigrationManager
         /// Updates the database to the latest version for the specified features.
         /// </summary>
         /// <param name="features">The features to be updated.</param>
-        Task UpdateAsync(IEnumerable<string> features);
+        [Obsolete("This method has been deprecated and moved into an extension method.")]
+        async Task UpdateAsync(IEnumerable<string> features) => await Task.WhenAll(features.Select(UpdateAsync));
```

Comment:
Do not use Task.WhenAll() for the same reason @Piedone mention above.

Reply:
So, how will they be optimized then?

Reply:
Not with parallel SQL queries :).